spirit/settings.py

```diff
@@ -33,6 +33,11 @@
 # followed by malicious HTML. See:
 # https://docs.djangoproject.com/en/1.11/topics/security/#user-uploaded-content
 ST_ALLOWED_UPLOAD_IMAGE_FORMAT = ('jpeg', 'gif')
+ST_ALLOWED_UPLOAD_FILE_FORMAT = (
```

Comment:
A comment mentioning only media types are allowed and a link to `https://www.iana.org/assignments/media-types/media-types.xhtml` would be handy. Maybe `ST_ALLOWED_UPLOAD_FILE_MEDIA_TYPE` would be a better name?